src/locales/de.json

```diff
@@ -2,14 +2,33 @@
 "3d_ie11_alert": "Sie verwenden InternetExplorer <= 11: Die 3D Darstellung wird nicht performant sein. Bitte nutzen Sie die neueste Version von Microsoft EDGE, Firefox oder Chrome um beste Ergebnisse zu erhalten",
 "3d_overlay_warning": "Wichtiger Hinweis: 2D-Daten in 3D.<br/>Sie betrachten jetzt Daten, die für die Darstellung in 3D nicht geeignet sind. Dies gilt insbesondere für Karten und Luftbilder. Diese werden in steilem Gelände bei der Nutzung in 3D verzerrt dargestellt\n",
 "3d_render_error": "Systemvorraussetzung für 3D Ansicht nicht erfüllt. Damit die 3D-Darstellung funktioniert, muss Ihr Gerät folgende Anforderungen erfüllen.Nutzen Sie die neueste Version von Microsoft EDGE, Firefox oder Chrome. Verwendung einer aktuellen Grafikkarte mit aktuellen Treibern. .",
+"Add layer": "Layer hinzufügen",
```

Comment:
As I understand, it's because ngeo does it like that?

The we should move them to the end of the files. And assure our translations really work. Don't we need them in empty.json? Or does ngeo take care of the translations?